components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/actions/ExtendWorkspaceTimeoutAction.kt

```diff
@@ -31,10 +31,10 @@ class ExtendWorkspaceTimeoutAction : AnAction() {
                     thisLogger().error("gitpod: failed to extend workspace timeout", e)
                 } else {
                     if (result.resetTimeoutOnWorkspaces.isNotEmpty()) {
-                        message = "Workspace timeout has been extended to three hours. This reset the workspace timeout for other workspaces."
+                        message = "Workspace timeout is extended to 180 minutes. This reset the workspace timeout for other workspaces."
```

Comment:
```suggestion
                        message = "Workspace timeout extended to 180 minutes. This reset the workspace timeout for other workspaces."
```

Reply:
> This reset the workspace timeout for other workspaces.

Is this still true? I assume workspace timeouts do not still apply just to one workspace, do they?

Reply:
Not sure about this. AFAIK, It only handles per workspace.